scripts/core/editor3/reducers/editor3.jsx

```diff
@@ -81,7 +81,7 @@ export const onChange = (state, newState, force = false) => {
     let contentChanged = state.editorState.getCurrentContent() !== newState.getCurrentContent();
 
     if (contentChanged || force) {
-        state.onChangeValue(editorState.getCurrentContent());
+        state.onChangeValue(editorState.getCurrentContent(), {plainText: !state.showToolbar});
```

Comment:
That makes sense, I guess I can just rename `showToolbar` everywhere

Reply:
Well not exactly, `showToolbar` would still exist as an input for Editor3 component, but coming from the logic of `plainText === true`

Reply:
hmmm I realized that we already set `showToolbar` based on `editorFormat` array. If the array is empty it's basically plainText, so I could use that logic that's already implemented.

Reply:
I've just taken a look at the code and it looks like we can rename `showToolbar` to `plainText`. As you noted, it is an input for Editor3 component, but we can rename that too because it's used only when plaintext is required. `const showToolbar = !props.singleLine && (props.editorFormat || []).length > 0;`